EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -1206,297 +1206,297 @@ QUEST_LV_0300_20160310_001205	What do we do with the useless scriptures?{nl}We u
-QUEST_LV_0300_20160310_001406	It's not a severe cut, but if we leave it, it will decay.{nl}There are many people hurt, but I can't do anything..
-QUEST_LV_0300_20160310_001407	Thank you. Thank you so much.{nl}This would help you a lot.
-QUEST_LV_0300_20160310_001408	I am sorry. I found a small box that was dropped on the wrecked ship..{nl}Do you know anything about it?{nl}
-QUEST_LV_0300_20160310_001409	Ahah.. So you don't know.{nl}This is sealed with magic so it can't be opened.{nl}
-QUEST_LV_0300_20160310_001410	Some important object seems to be inside..{nl}Can you help me opening the box?
+QUEST_LV_0300_20160310_001394	It's embarassing as mayor, but I once thought to give up on everything.{nl}Whatever it is, I think we made the right decision to stand still. 
+QUEST_LV_0300_20160310_001395	A large number of monsters came? {nl}Damn it... The effects seemed good, but we will be in danger.{nl}
+QUEST_LV_0300_20160310_001396	I am so sorry to give you a hard task.{nl}I will tell the other people not to touch the pocket.{nl}
+QUEST_LV_0300_20160310_001397	That coral... Where did you get it?{nl}That's called Arnim. It is a very important coral used in medicine.
+QUEST_LV_0300_20160310_001398	There must be a habitat near the place where you obtained it...{nl}
+QUEST_LV_0300_20160310_001399	I am sorry, but can you look for their habitat?{nl}If you could look for the habitat of the Arnim corals, it would help the villagers a lot.
+QUEST_LV_0300_20160310_001400	If you could find the habitat of the corals of Arnim, it will help the villagers a lot.
+QUEST_LV_0300_20160310_001401	Nugara Sandbank?{nl}That deep... We will have hard time extracting it.{nl}
+QUEST_LV_0300_20160310_001402	First, we better let the refugees know it. We should find a way together.{nl}Thank you so much!
+QUEST_LV_0300_20160310_001403	I am sorry, but can you help us out?{nl}We have many wounded people, but the medicine ran out... {nl}It's becoming more severe if we don't get any help...{nl}
+QUEST_LV_0300_20160310_001404	We've already used all the medicine which the merchants had {nl}and we have no way of getting the medicine again.{nl}We can make an ointment if we have the ingredients...
+QUEST_LV_0300_20160310_001405	Please, get me Greentoshell Shell Fragments and Blue Lapasape Mosses.{nl}With those I can make an emergency ointment for the injuries.
+QUEST_LV_0300_20160310_001406	It's not a severe cut, but if we leave it, it will decay.{nl}There are many people hurt, but I can't do a thing...
+QUEST_LV_0300_20160310_001407	Thank you. Thank you so much.{nl}This will help you a lot.
+QUEST_LV_0300_20160310_001408	I am sorry. I found a small box that was dropped on the wrecked ship...{nl}Do you know anything about it?{nl}
+QUEST_LV_0300_20160310_001409	Ahah... So you don't know.{nl}This is sealed with magic so it can't be opened.{nl}
+QUEST_LV_0300_20160310_001410	Some important object seems to be inside...{nl}Can you help me opening the box?
```

Comment:
"Some important object" -> "Something important"

"Can you help me opening the box?" -> "Can you help me open the box?"